doc/examples/scraper/README.md

```diff
@@ -1,6 +1,6 @@
 # Quick Start Guide: Web Scraper
 In this guide you're going to create a Pachyderm pipeline to scrape web pages. 
-We'll use a standard unix tool, `wget`, to do our scraping. 
+We'll use a standard unix tool, `wget`, to do our scraping.
```

Comment:
I agree on having company policy for this stuff. I'm going to put together a document on it. I'm not sure what the guidelines should be in this case though given that the file in question is prose. Will people be ok with having the guidelines on prose just be "follow standard English style." In my mind that precludes things like trailing spaces and obviously none english things like that. But in terms of interesting English stylistic things I don't think there's much need to be prescriptive since we just won't run into them too much. Does that seem reasonable to people?